lldb/source/Plugins/ObjectFile/COFF/ObjectFileCOFF.cpp

```diff
@@ -271,9 +271,9 @@ void ObjectFileCOFF::ParseSymtab(lldb_private::Symtab &symtab) {
     const auto COFFSymRef = m_object->getCOFFSymbol(SymRef);
 
     Expected<StringRef> NameOrErr = SymRef.getName();
-    if (auto error = NameOrErr.takeError()) {
-      LLDB_LOG(log, "ObjectFileCOFF: failed to get symbol name: {0}",
-               llvm::fmt_consume(std::move(error)));
+    if (!NameOrErr) {
+      LLDB_LOG_ERROR(log, NameOrErr.takeError(),
+                     "ObjectFileCOFF: failed to get symbol name: {0}");
```

Comment:
I don't think you need to specify `ObjectFileCOFF:` anymore since `LLDB_LOG_ERROR` will take care of adding the file name and the function symbol for you.

Reply:
Hey Ismail! Right, this seems unconventional compared to other logging code in LLDB. It's not something that changed when moving from `LLDB_LOG` to `LLDB_LOG_ERROR` though. We should keep printing the same info, but I agree that this is worth fixing while I am here.